### PR TITLE
Update Link Treatment to Match 10up.com

### LINKED ISSUE #266: Update Link Treatment to Match 10up.com
Update links on the best practice site to match 10up.com: https://10up.com/blog/ (link, hover, action, focus)

### PR DESCRIPTION
**Pull request by colorful-tones**
Addresses and fixes #266 

I targeted the links in the main content area to match the treatment currently seen on https://10up.com/blog/

I also did some clean up (alpha sorting, replace static hex colors with variables) in the `_header.scss` and `_footer.scss` partials.

I also made sure the header and footer links were not affected by the new styles.

Attaching a Gif of resulting treatment

10upengineering-link-treatment

### CODE REVIEW

**Review by timwright12 — CHANGES_REQUESTED** [review]
Looks like the homepage links are quite right, the background gradient is pushed down a bit, the hover is moving the link a bit, and there's a text-decoration set. Let's just remove the treatment for these links, it looks like there might be a contrast issue anyway. We can just go with normal black links, underline on hover here.

screen shot 2019-01-18 at 9 16 03 am

**Comment by colorful-tones** [comment]
I updated the Table of Contents links. Thanks Tim!

File: _includes/_sass/includes/_table-of-contents.scss [hunk 1]
```diff
@@ -74,13 +74,14 @@
...
 
 	a {
+		background-image: none;
+		color: $almost-black;
 		display: inline-block;
-		padding-bottom: 1px;
+		text-decoration: none;
 	}
 
 	a:hover {
```
Comment (timwright12) [inline _includes/_sass/includes/_table-of-contents.scss:83]: @colorful-tones remember to set :focus styles as well.

**Review by timwright12 — CHANGES_REQUESTED** [review]

File: _includes/_sass/includes/_table-of-contents.scss (hunk 1 above)
Comment (colorful-tones) [reply _includes/_sass/includes/_table-of-contents.scss:83]: I added :focus styling. Thanks!

File: _includes/_sass/includes/_footer-about.scss [hunk 2]
```diff
@@ -31,6 +37,15 @@
 			font-size: 1rem;
 			margin: 1rem 0;
 		}
+
+		a {
+			text-decoration: underline;
+
+			&:active,
```
Comment (timwright12) [inline _includes/_sass/includes/_footer-about.scss]: `&:focus` rather than `&:active`

File: _includes/_sass/includes/_footer.scss [hunk 3]
```diff
@@ -1,62 +1,70 @@
...
+		padding-left: $header-logo-large;
 		vertical-align: middle;
 		width: 49%;
-		padding-left: $header-logo-large;
 
 		a {
 			color: lighten($color-medium-dark, 10%);
 			padding: .5em;
 
 			&:hover {
```
Comment (timwright12) [inline _includes/_sass/includes/_footer.scss:67]: Add `&:focus` styles

File: _includes/_sass/includes/_footer.scss [hunk 4]
```diff
@@ -65,40 +73,42 @@ footer[role="contentinfo"] {
...
 		a {
-			display: inline-block;
 			color: $color-dark-text;
+			display: inline-block;
 			padding: 21px 0 18px;
-			// border-bottom: 3px solid transparent;
-
 			position: relative;
 
 			&:hover {
```
Comment (timwright12) [inline _includes/_sass/includes/_footer.scss:83]: Add `&:focus` styles

File: _includes/_sass/includes/_header.scss [hunk 5]
```diff
@@ -98,85 +99,104 @@
...
+	background-image: none;
 	display: block;
 	font-size: 1.3rem;
-	background-color: black;
 
 	@include breakpoint($breakpoint-bravo) {
 		display: none;
 	}
+
+	&:hover {
```
Comment (timwright12) [inline _includes/_sass/includes/_header.scss:122]: Add `&:focus` styles

File: _includes/_sass/includes/_header.scss [hunk 6]
```diff
@@ -98,85 +99,104 @@
...
 	display: block;
 	font-size: 1.3rem;
-	background-color: black;
 	right: 2.5em;
 
 	@include breakpoint($breakpoint-bravo) {
 		right: 0;
 	}
+
+	&:hover {
```
Comment (timwright12) [inline _includes/_sass/includes/_header.scss:146]: Add `&:focus` styles

File: _includes/_sass/includes/_header.scss [hunk 7]
```diff
@@ -190,40 +210,45 @@
...
-		color: $color-dark-text;
+		padding: .8rem 0;
 
 		@include breakpoint($breakpoint-bravo) {
-			padding: 21px 0;
 			font-size: 11px;
+			padding: 21px 0;
 			position: relative;
 
 			&:hover {
```
Comment (timwright12) [inline _includes/_sass/includes/_header.scss:234]: Add `&:focus` styles

File: _includes/_sass/base/_base.scss [hunk 8]
```diff
@@ -33,32 +33,26 @@ img {
...
 	text-decoration: none;
+	transition: all .1s;
 
 	&:visited {
-		color: desaturate($color-accent-alternate, 20%);
+		color: $color-dark;
 	}
 
 	&:hover,
 	&:active {
```
Comment (timwright12) [inline _includes/_sass/base/_base.scss]: `&:focus` rather than `&:active`

File: _includes/_sass/includes/_header.scss [hunk 9]
```diff
@@ -98,85 +99,104 @@
...
 	@include breakpoint($breakpoint-bravo) {
 		display: none;
 	}
+
+	&:hover {
+		color: $color-accent-alternate-2;
+	}
 }
 
 .main-nav-supplemental-icon {
```
Comment (timwright12) [inline _includes/_sass/includes/_header.scss:127]: The `:visited` state for this link is washing out the color, let's make sure that doesn't happen.

**Review by timwright12 — CHANGES_REQUESTED** [review]

File: _includes/_sass/includes/_footer-about.scss (hunk 2 above)
Comment (colorful-tones) [reply _includes/_sass/includes/_footer-about.scss]: Fixed.

File: _includes/_sass/includes/_footer.scss (hunk 3 above)
Comment (colorful-tones) [reply _includes/_sass/includes/_footer.scss:67]: Fixed

File: _includes/_sass/includes/_header.scss (hunk 5 above)
Comment (colorful-tones) [reply _includes/_sass/includes/_header.scss:122]: Fixed

File: _includes/_sass/includes/_footer.scss (hunk 4 above)
Comment (colorful-tones) [reply _includes/_sass/includes/_footer.scss:83]: Fixed

File: _includes/_sass/includes/_header.scss (hunk 6 above)
Comment (colorful-tones) [reply _includes/_sass/includes/_header.scss:146]: Fixed

File: _includes/_sass/includes/_header.scss (hunk 7 above)
Comment (colorful-tones) [reply _includes/_sass/includes/_header.scss:234]: Fixed

File: _includes/_sass/base/_base.scss (hunk 8 above)
Comment (colorful-tones) [reply _includes/_sass/base/_base.scss]: Fixed

File: _includes/_sass/includes/_header.scss (hunk 9 above)
Comment (colorful-tones) [reply _includes/_sass/includes/_header.scss:127]: Fixed

**Comment by colorful-tones** [comment]
@timwright12 thanks for all the great catches on those :focus and :visited updates. Those should all be fixed now.

**Review by timwright12 — APPROVED** [review]

**Pull request merged**